synthetic_data/distinct_generators/int_generator.py

```diff
@@ -4,20 +4,22 @@
 
 
 def random_integers(
-    rng: Generator, min_value: int = -1e6, max_value: int = 1e6, num_rows: int = 1
+    rng: Generator, min: int = -1e6, max: int = 1e6, num_rows: int = 1
 ) -> np.array:
     """
     Randomly generates an array of integers between the given min and max values.
 
     :param rng: the np rng object used to generate random values
     :type rng: numpy Generator
-    :param min_value: the minimum integer that can be returned
-    :type min_value: int, optional
-    :param max_value: the maximum integer that can be returned
-    :type max_value: int, optional
+    :param min: the minimum integer that can be returned
+    :type min: int, optional
+    :param max: the maximum integer that can be returned
+    :type max: int, optional
     :param num_rows: the number of rows in np array generated
     :type num_rows: int, optional
 
     :return: np array of integers
     """
-    return rng.integers(min_value, max_value, (num_rows,))
+    max += 1
```

Comment:
```suggestion
    # because `rng.integers` has an exclusive `max`, we need to ensure the max of the data is 
    # `n-1` the max parameter value. 
    max += 1
```

Reply:
something to this effect